Gemfile.lock

```diff
@@ -78,17 +78,17 @@ GEM
       regexp_parser (~> 1.5)
       xpath (~> 3.2)
     childprocess (3.0.0)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
+    crass (1.0.6)
```

Comment:
### [crass](https://github.com/rgrove/crass/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.5 → 1.0.6

[_[change-log](https://github.com/rgrove/crass/blob/v1.0.6/HISTORY.md), [source-code](https://github.com/rgrove/crass/tree/v1.0.6)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rgrove/crass/compare/v1.0.5...v1.0.6).

These are the individual commits:
- (b349cc8) [Add project metadata to the gemspec](https://github.com/rgrove/crass/commit/b349cc8299a801a6f93dd8ea5613cce9b7e688d6)
- (fbedb91) [Update history](https://github.com/rgrove/crass/commit/fbedb916e37c70dddd90f6aa51aa61953b9d54d9)
- (3e02596) [Limit number values to a sensible range](https://github.com/rgrove/crass/commit/3e025966ccbb2fdb190a1b4c3c4542109dc72fa7)
- (87f642e) [Release 1.0.6](https://github.com/rgrove/crass/commit/87f642e80670dc1093eddf4d3a93cbd40a9551c6)


</details>